### PR TITLE
Added initialization of the project , add and Remove functionality

### PR DESCRIPTION
**Pull request by moise10r**
In this PR:
- I Initialised React app.
- I added React-Redux.
- I structure my application files using a ducks pattern for your Redux files. 
- I added routing between components

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <App />
+    </BrowserRouter>
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById('root'),
 );
```
Comment (arslanbisharat) [inline src/index.js]: - [x] Please make sure the index has 2 `<Link>`s in the navbar which leads to the booklist and categories locations.
The navbar might be in its own component, that's ok!


- [x] Please make sure the index has 2 `<Route>`s, one for the BookList component and the other for the Categories component. In order to not show a blank page when visiting the site for the first time, the BookList component should be the default page.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please remove all unnecessary files from your project to make it clean. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.js (hunk 1 above)
Comment (moise10r) [reply src/index.js]: I have put the route in the App.js file and I took it as the entry point of the Application
book

**Comment by arslanbisharat** [comment]
Please make the changes as requested

**Comment by Davidosky007** [comment]
## Hi @moise10r 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**